src/main/kotlin/com/progreizh/animaliste/AnimalisteApplication.kt

```diff
@@ -15,7 +15,7 @@ class AnimalisteApplication{
             override fun addCorsMappings(registry: CorsRegistry) {
                 registry.addMapping("/**")
                     .allowedOrigins("http://localhost:8080")
-                    .allowedMethods("GET", "PUT", "UPDATE", "DELETE")
+                    .allowedMethods("GET", "PUT", "UPDATE", "DELETE", "POST")
```

Comment:
```suggestion
                    .allowedMethods("GET", "PUT", "DELETE", "POST")
```
UPDATE n'existe pas j'étais bourré ?